back-end/src/sciler/config/workingConfigTypes.go

```diff
@@ -166,10 +166,10 @@ func (r *Rule) Finished() bool {
 // Execute performs all actions of a rule
 func (r *Rule) Execute(handler InstructionSender) {
 	for _, action := range r.Actions {
-		action.Execute(handler)
+		go action.Execute(handler)
```

Comment:
Because of this goroutine, the second goroutine in handler_helper.go line 52 is not needed

Reply:
removed